python/fastquant/indicators/custom.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Import standard library
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+from pkg_resources import resource_filename
+import datetime
+import sys
+
+# Import modules
+import backtrader as bt
+
+
+class CustomIndicator(bt.Indicator):
+
+    """
+    Custom Indicator
+    """
+
+    lines = ("custom",)
+
+    params = (
+        ("custom_column", "custom"),
+    )
+
+    plotlines = dict(
+        sentiment=dict(
```

Comment:
I think this won't generate since there is no `sentiment` in `lines`

Reply:
Great catch!

Reply:
Will just take out the line specific plot. I'm already happy with the current plot :smile:

Reply:
Took out the line specific plotting parameters since the object level plot looks okay to me :)